models/articles.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection");
+
+exports.fetchArticle = (article_id) => {
+  let queryString = "SELECT * FROM articles WHERE article_id = $1";
+  return db.query(queryString + ";", [article_id]).then(({ rows }) => {
```

Comment:
You don't need to add the ; here. The SQL query will work without it when we're using nodePostgres 🙂